### PR TITLE
Update emojis

### PR DESCRIPTION
**Pull request by damassi**
Updates emojis to be consistent across actions: 

screen shot 2019-01-22 at 1 12 37 pm

### CODE REVIEW

File: slack-buggybot/commands/next.rb [hunk 1]
```diff
@@ -28,16 +28,20 @@ def self.call(client, data, match)
...
         when 'verified'
           fate = 'verified'
           message = 'verified the existence of'
+          emoji = ':110:'
         when 'interlinked'
           fate = 'interlinked'
           message = 'added context to'
-        when 'none'
+          emoji = ':link:'
+        when 'skip'
```
Comment (damassi) [inline slack-buggybot/commands/next.rb:44]: `@buggy next skip` seems more intuitive than `@buggy next none`

File: slack-buggybot/models/bug.rb [hunk 2]
```diff
@@ -27,7 +27,7 @@ def self.in_event(event_id)
     end
 
     def self.user_existing_bug(user_id:, event_id:)
-      Bug.in_event(event_id).where(completed: nil).where(assignee: user_id).first
+      Bug.done_in_event(event_id).where(completed: nil).where(assignee: user_id).first
```
Comment (damassi) [inline slack-buggybot/models/bug.rb:30]: Not sure how to run / test this locally!

Comment (ashfurrow) [reply slack-buggybot/models/bug.rb:30]: No worries, I can test it later 👍

File: slack-buggybot/commands/next.rb (hunk 1 above)
Comment (ashfurrow) [reply slack-buggybot/commands/next.rb:44]: Yup, makes sense. There’s one mention of none left in the readme but I can fix after.

**Review by ashfurrow — APPROVED** [review]
Looks good, let me do the deploy thing next.

**Pull request merged**